secret.py

```
@@ -0,0 +1,14 @@
+def log_in(password):
+    conn = sqlite3.connect('users.db')
```

Ruff (0.8.2)

Undefined name `sqlite3`

(F821)